apps/web/src/design-system/dropdown/Dropdown.styles.ts

```diff
@@ -26,5 +25,9 @@ export default createStyles((theme: MantineTheme) => {
     itemHovered: {
       backgroundColor: dark ? colors.B30 : colors.B98,
     },
+    divider: {
+      margin: '10px',
+      borderTop: `0.0625rem solid ${colors.B30}`,
```

Comment:
If this one is the divider color here, we need to add 'Light BG' for the light them.

Reply:
yes, thanks for pointing on that :)